environment/values.yaml

```diff
@@ -7,6 +7,7 @@ env:
 repository:
   url: YOUR_INFRA_REPO_URL
   charts: argocd/charts/infra
+  app_charts: argocd/charts/app
   envs: argocd/environments
   envs_from_charts: ../../../environments
   revision: master
```

Comment:
will it look like this?:
chart_apps: 
  argocd:
     enabled: true
     app: true

Reply:
yeah so something like this
```
   some_app2:
     enabled: true
     namespace: some_app2
-    chart: argocd/charts/app/some_app2
+    app: true
   rabbitmq:
     enabled: true
     namespace: rabbitmq
@@ -41,7 +41,7 @@ chart_apps:
   some_app1:
     enabled: true
     namespace: some_app1
-    chart: argocd/charts/app/some_app1
+    app: true
```